### ISSUE TITLE
Dependency on pixman code

### ISSUE THREAD

**Issue by suve** [LibreSprite/LibreSprite#28]
When compiling with `-DUSE_SHARED_PIXMAN=TRUE`, the build fails due to some code depending on `pixman`'s internal code. 
```
In file included from ../src/doc/blend_funcs.cpp:23:0:
../src/./doc/blend_internals.h:11:10: fatal error: ../../third_party/pixman/pixman/pixman-combine32.h: No such file or directory
 #include "../../third_party/pixman/pixman/pixman-combine32.h"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Since `pixman` source code is required even when using a shared library, maybe it should be moved to `src`? At the very least, the documentation should be updated to clearly state this.

LibreSprite version: 0a01bb9a9d6cf65868cebc7f3df346044a6b1534
OS: Fedora 27

**Comment by nanoslayer** [comment]
This was fixed in #102. This issue can be closed.

**Comment by Zughy** [comment]
Huh, right, thanks nanoslayer. Closing
